karton/classifier/classifier.py

```diff
@@ -84,11 +132,19 @@ def wrapper(content, mime):
 
     def process(self, task: Task) -> None:
         sample = task.get_resource("sample")
-        sample_class = self._classify(task)
+
+        sample_classes = []
+
+        if self.yara_rules:
+            sample_classes += self._classify_yara(task)
+
+        filemagic_classification = self._classify(task)
```

Comment:
nitpick, but since we now have two methods calling them `_classify` and `_classify_yara` is a bit weird. Maybe change this to `_classify_using_python` / `_classify_using_yara` or `_classify_yara` / `_classify_filemagic` or something :thinking: Not sure about this, if you don't have a good idea how to solve this feel free to ignore this comment.

Reply:
:ack: